package.json

```diff
@@ -73,6 +73,9 @@
     "react": "^16.3.0",
     "shallow-equal": "^1.1.0"
   },
+  "peerDependencies": {
```

Comment:
Since it’s a dev dep, is peer dep really needed? We don’t need it in production.